### DISCUSSION TITLE
Optimierungen Bereich technische Daten

### DISCUSSION THREAD

**Discussion by moritzvolkmar** [Samedis-care/samedis-care-community#6] [Idea retired / denied]
1. Mehrfachauswahl Netzwerkkonnektivität
![Bildschirmfoto 2022-05-31 um 15 13 28](https://user-images.githubusercontent.com/80519295/171606872-62cb6297-f30e-43d1-a4dd-98d6b04bc9fd.png)

2. Infofelder i hinzufügen was man wo wie eintragen sollte
![Bildschirmfoto 2022-05-31 um 15 14 35](https://user-images.githubusercontent.com/80519295/171606994-67febd54-49ed-4d59-a968-eb7a943b8ede.png)

**Comment by YvesR** [comment]
1. war angedacht, was genutzt wird, nicht was das Gerät hat, bei USP Ports ist das ja dann auch so.
2. Wer definiert die Texte? Dann können wir das gerne einfügen.

**Reply by moritzvolkmar** [reply]
1. auch das können durchaus mehrere sein. 
2. Das mache ich

**Comment by moritzvolkmar** [comment]
@YvesR Hier ist aktuell auch ein kleiner Logikfehler enthalten. 
![Bildschirmfoto 2022-09-30 um 11 27 02](https://user-images.githubusercontent.com/80519295/193240070-055baa48-f7ab-4c12-9f84-8e1eb29ca8f1.png)

**Comment by YvesR** [comment]
- `Network connectivity` > wird korrigiert, so dass man löschen kann oder keine wählen.
- Prüfung Umbau auf multiselect

**Comment by YvesR** [comment]
Update: No multi select, what is connected in use is selected here. The available options will be recorded on the type in the future.
